src/main/java/quickndirty/minisurveymonkey/WebController.java

```diff
@@ -0,0 +1,15 @@
+package quickndirty.minisurveymonkey;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class WebController {
+
+    @GetMapping("/create-survey")
```

Comment:
Do we want to put this route as the default page. Or are we planning on creating some home page later.

Reply:
I was thinking the default page should probably be a list of all the surveys you could answer, all the surveys closed that you can look at the results for, plus a link to create your own somewhere, so we would need another route if we wanted to go that way.